packages/pages/src/vite-plugin/build/closeBundle/manifest.ts

```diff
@@ -65,6 +67,50 @@ export const generateManifestFile = (
   fs.remove(path.resolve(distPath.path, "manifest.json"));
 };
 
+/**
+ * Scans for paths in dist/assets/client/ and finds the client template's
+ * path and feature name.
+ * @param projectStructure
+ * @returns Promise<string[][]> with the feature name and path
+ */
+const getClientPaths = async (
```

Comment:
Why is this whole function more complicated than how we get the renderPaths above? Couldn't you just turn that code instead a generic function that just asks for the asset folder?

```
function getAssetBundlePaths(assetPath) {
const filePaths = glob.sync(
    convertToPosixPath(
      path.resolve(
        projectStructure.config.rootFolders.dist,
        projectStructure.config.subfolders.assets,
        assetPath,
        "**/*.js"
      )
    )
  );
  return filePaths.map((filepath) => [
    path.parse(filepath).name.split(".")[0], // get the name of the file without the hash or extension
    convertToPosixPath(distPath.getRelativePath(filepath)),
  ]);
```

Reply:
Can you make a common function like I mentioned?

Reply:
Oh yes added it to client.ts